docs/Changelog.md

```diff
@@ -10332,6 +10332,54 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain input and output types to integer tensors.</dd>
 </dl>
 
+### <a name="GatherND-11"></a>**GatherND-11**</a>
+
+  Given `data` tensor of rank r >= 1, and `indices` tensor of rank q >= 1, gathers
```

Comment:
Is it possible to have `q>r`?

Reply:
This is a very good question - thanks. I have addressed all this in the doc for this op in my latest commits.